### DISCUSSION TITLE
How to bind an input dynamically based on day?

### DISCUSSION THREAD

**Discussion by RomanMunar** [mProjectsCode/obsidian-meta-bind-plugin#149] [Q&A] [answered]
I'm looking to edit today's daily note's frontmatter on another note.

**Comment by mProjectsCode** [comment]
you would need to have templater set the file name of the input field on note creation

**Reply by RomanMunar** [reply]
i want the date to change automatically to today's date. seems like i would have to use dataviewjs and order the rendering of DV.date & plug it in meta bind. is there an example i could use for making this?

**Comment by RomanMunar** [comment, marked as answer]
🎉🎉🎉

````
```dataviewjs
const today = dv.date('today').toISODate()~~toFormat('yyyy-MM-dd')~~

dv.paragraph('```meta-bind' + `
INPUT[text:${today}#test]
`+'```')
```
````

**Reply by Nitero** [reply]
is there a way to do the same thing for a button in a dataviewjs query?

I can get it to work with a toggle, but the button seems impossible because you cannot define it inline and if you define it outside then the bindTarget is fixed to the current note?

**Reply by RomanMunar** [reply]
You can open a ticket for that, title could be "An Option for Buttons to perform actions on other files"
I haven't tried it for buttons either but a viable alternative is to create the buttons beneath a heading on the file you want to perform actions on then query/embed that heading with dataview/![[]] unto a separate note.

**Reply by Nitero** [reply]
Im not sure thats a better solution than just using a toggle... if you have that button on a lot of notes changing it later will be a pain

Ive create a [new issues](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/issues/259) as suggested
